.gitlab/llnl-ci-conda.yml

```diff
@@ -0,0 +1,117 @@
+stages:
+  - env-setup
+  - test
+  - cleanup
+
+.final_steps:
+  script: &finalize
+    - chgrp -fR exaworks $SDK_WORK_DIR/conda || true
+    - chmod -fR 02770 $SDK_WORK_DIR/conda    || true
+
+conda-env-setup:
+  stage: env-setup
+  parallel:
+    matrix:
+      - HOST : [ "quartz", "ruby"]
+        RUNNER_TYPE: "shell"
+        # - HOST : [ "lassen" ]
+        #   ENVIRONMENT: [ "BATCH" ]
+        #   RUNNER_TYPE: "batch"
+  tags:
+    - $HOST
+    - $RUNNER_TYPE
+  script:
+    - mkdir -p ${SDK_WORK_DIR}/conda/
+    - cd ${SDK_WORK_DIR}/conda/
+    - eval "$(conda shell.posix hook)"
+    - conda create --prefix ./${HOST}-exaworks-sdk-venv python=3.7 -y
```

Comment:
let's keep env name as part of `matrix` per host
```
  parallel:
     matrix:
       - HOST: [ "quartz" ]
         CONDA_ENV_NAME: [ "quartz-env" ]
       - HOST: [ "lassen" ]
         CONDA_ENV_NAME: [ "lassen-env" ]
       - HOST: [ "ruby" ]
         CONDA_ENV_NAME: [ "ruby-env" ]
```
or just reuse env name as a host name (all of them are inside conda-dir, which is inside sdk-dir, thus hierarchy is preserved)

Reply:
Now addressed